### PR TITLE
[rpm] Integrate libev 4.33. JB#50106

### PR DESCRIPTION
**Pull request by rainemak**

### CODE REVIEW

**Review by mkosola — APPROVED** [review]

File: libev.spec [hunk 1]
```diff
@@ -0,0 +1,84 @@
+Name:             libev
+Version:          4.33
+Release:          1
+Summary:          High-performance event loop/event model with lots of features
+
+License:          BSD or GPLv2+
+URL:              http://software.schmorp.de/pkg/libev.html
+Source0:          %{name}-%{version}.tar.gz
```
Comment (okodron) [inline libev.spec:8]: Looks like `libev-4.20-Respect-the-CFLAGS-if-defined.patch` is not applied any more since 4.23.

File: libev.spec [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+%{_includedir}/ev++.h
+%{_includedir}/ev.h
+%{_libdir}/%{name}.so
+%{_mandir}/man?/*
+
+%files libevent-devel
+%{_includedir}/event.h
+
+%files doc
+%license LICENSE
```
Comment (okodron) [inline libev.spec:83]: Already in the main package.

Comment (rainemak) [reply libev.spec:83]: Yes but license should still be here on the doc as it is standalone.

File: libev.spec (hunk 1 above)
Comment (rainemak) [reply libev.spec:8]: Good that you checked. Leftover also in fedora then. :)
Better not to add it here either.

**Review by mkosola — COMMENTED** [review]
LGTM

**Pull request merged**